css/queries.css

```diff
@@ -0,0 +1,154 @@
+/* Media queries */
+@media (min-width: 48em) {
+  .all {
```

Comment:
- [ ] Kindly make sure the page is 100% responsive on all screen sizes.
As you can see how your page looks messy on `768px` screen size.

![image](https://user-images.githubusercontent.com/97100091/200888366-b7c88334-ce42-4f7f-bdc8-4c4143d9f5fc.png)